wkambale/missingpersons
# Add an analytics page

# Adds analytics to data
1. I've created a new file. `analytics.html`
1. I read the data file and run analytics by gender, status and holding location.

## To test
At the bottom where there's privacy policy, click on the analytics link.
You'll also be able to navigate back to the home page from the analytics page

Expect to see something like this
<img width="1366" alt="Screenshot 2024-07-23 at 20 36 32" src="https://github.com/user-attachments/assets/5c3d123b-fc22-4383-aa1a-b77ff766a82a">
